README.md

```diff
@@ -75,19 +75,18 @@ details please see the [Contribution Guidelines](CONTRIBUTING.md).
 
 ## License
 
-Artistic or creative content is licensed under the *Creative Commons
-Attribution-Share Alike 3.0 Unported* license
-(http://creativecommons.org/licenses/by-sa/3.0/).
+The FreeOrion source code is licensed under the terms of [GPL v2],
+game assets are licensed under the terms of [CC-BY-SA-3.0].  For more details
+please see the [License File](default/COPYING).
 
-This includes artwork, music, sounds, text that determines how art, music or
-sound is played or displayed in game, game content such as technologies,
-buildings, specials and their associated in-game text, and the scripts that
-determine their in-game functionality.
+Additional to the immediate project sources the FreeOrion source tree bundles
+some third party projects or assets which may be licensed under different
```

Comment:
Well, those third party assets/libraries are already under a different but compatible license, why should we license them also under *our* license?